rt_gene/src/rt_gene/tracker_generic.py

```diff
@@ -10,11 +10,11 @@
 
 
 class TrackedSubject(object):
-    def __init__(self, box, face, landmarks, marks):
+    def __init__(self, box, face, transformed_eye_landmarks, landmarks):
         self.box = box
         self.face_color = face
-        self.transformed_landmarks = landmarks
-        self.marks = marks
```

Comment:
I think you missed a use of `marks` in L63 of estimate_gaze_standalone.py.

Reply:
Done!